assets/js/modules/ads/components/common/WooCommerceRedirectModal.js

```diff
@@ -130,11 +157,19 @@ export default function WooCommerceRedirectModal( {
 		onDismiss,
 		navigateTo,
 		googleForWooCommerceRedirectURI,
+		viewContext,
+		trackEventLabel,
 	] );
 
 	const onSetupCallback = useActivateModuleCallback( 'ads' );
 
-	const onContinueWithSiteKit = useCallback( () => {
+	const onContinueWithSiteKit = useCallback( async () => {
+		await trackEvent(
```

Comment:
Continuing with SK doesn't trigger a navigation, does it? I think it pushes a change in the URL but doesn't actually reload the page, no?

Reply:
@aaemnnosttv  I am not actually sure, I will test this now.

Reply:
@aaemnnosttv this is indeed the case, not actual navigation physically, so I have reverted the `async` in my latest iteration.